package.json

```diff
@@ -4,13 +4,20 @@
   "description": "Create a V8 heap snapshot when an \"Out of Memory\" error occurs, or create a heap snapshot or CPU profile on request.",
   "main": "index.js",
   "scripts": {
+    "install": "node-pre-gyp install --fallback-to-build",
```

Comment:
When would we need to call this npm script?

Reply:
This is how users of `node-oom-heapdump` get the prebuilt binary installed for them. The script is [run by npm](https://docs.npmjs.com/misc/scripts) after the package is installed, so the it doesn't have to run `node-gyp` afterwards.

This approach is documented by `node-pre-gyp` [here](https://github.com/mapbox/node-pre-gyp#1-add-new-entries-to-your-packagejson).

Reply:
(Just realised I didn't actually answer your question) -- nobody calls this script manually, as it's invoked automatically by npm. Both developers and end-users of `node-oom-heapdump` benefit from this change.

If a developer is changing `node_oom_heapdump_native.cc` they will need to make sure they change the `package.json` version, otherwise `node-pre-gyp` will be helpful and download the prebuilt version that corresponds to the version number in `package.json`. Perhaps this could be solved by bumping the version number automatically after every release?

Reply:
Ah yes, of course. "install" is the script that by default will run when calling `npm install`, (e.g. `npm run install`).
The version will always be updated when publishing a release, so that shouldn't be an issue.

For local development, we could opt for a npm script that will automatically update the npm version, something like `npm version patch`, so the prebuilt binaries are bypassed.